docs/reports/2026-05-03-issue-466-cooking-procedural-inventory.md

```
@@ -45,20 +51,111 @@ These reuse the existing `CookingEffectFragmentTranslator` max-HP details rule:
+quickness, MA, to-hit, XP, natural healing, random stat, and triggered stat
+detail rows.
+
+## Dictionary Cleanup
+
+The following broad/prefix-like exact rows were removed from
+`world-effects-cooking.ja.json` and guarded by
+`LocalizationCoverageTests.DynamicProducerRoutes_DoNotKeepKnownConcreteExactKeys`:
+
+- `@they get +`
+- `@they expel quills per the Quills mutation at level`
+- `@they expel quills per the Quills mutation at level `
```

markdownlint-cli2 (0.22.1)

[warning] Spaces inside code span elements

(MD038, no-space-in-code)